src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs

```diff
@@ -364,13 +363,20 @@ private static void CreatePlaneTessellation(IRenderPackage package, Plane plane)
             var nEnd = plane.Origin.Add(plane.Normal.Scale(2.5));
             package.AddLineStripVertex(nEnd.X, nEnd.Y, nEnd.Z);
 
-            //Add the line vertex for the plane line geometry (4 plane edges and 1 normal).
+            //Add the line vertex data for the plane line geometry (4 plane edges and 1 normal).
             for (var i = 0; i < 5; i++)
             {
                 package.AddLineStripVertexCount(2);
                 package.AddLineStripVertexColor(MidTone, MidTone, MidTone, 255);
                 package.AddLineStripVertexColor(MidTone, MidTone, MidTone, 255);
             }
+
+            //dispose helper geometry
+            a.Dispose();
+            b.Dispose();
+            c.Dispose();
+            d.Dispose();
+            cs.Dispose();
```

Comment:
How about using `using` statements instead?

Reply:
I tried this and it was more convoluted then the dispose.  I think just keep it simple here.